src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/JurisdictionToUserMapping.java

```diff
@@ -17,13 +18,14 @@ public void setUsers(Map<String, Map<String, String>> users) {
             .entrySet()
             .stream()
             .map(this::createPair)
-            .collect(toMap(Pair::first, Pair::second));
+            .collect(toMap(Entry::getKey, Entry::getValue));
     }
 
-    private Pair<String, Credential> createPair(Map.Entry<String, Map<String, String>> entry) {
+    private Entry<String, Credential> createPair(Entry<String, Map<String, String>> entry) {
         String key = entry.getKey().toLowerCase();
         Credential cred = new Credential(entry.getValue().get("username"), entry.getValue().get("password"));
-        return new Pair<>(key, cred);
+
+        return new AbstractMap.SimpleEntry<>(key, cred);
```

Comment:
hmmm  ... a bit terse in construction but I guess the getKey and getValue out weigh this little bit of terseness